src/form/validated-form.tsx

```diff
@@ -87,6 +86,18 @@ ValidatedForm.displayName = 'ValidatedForm';
 export interface ValidatedInputProps extends InputProps {
   // name of the component, also used for validation
   name: string;
```

Comment:
`ValidatedInputProps` extends `InputProps` which should already have these defined, the same goes for the next interface as well. What is the issue that you are seeing?

Reply:
Hi, I thought it was strange too, I saw that you extended the properties, but for some reason the application doesn't see these properties.
Image in chrome terminal: [https://postimg.cc/Hc0PTwft](url)

And we are also having problems building the application with ./mvnw:
Image in terminal: [https://postimg.cc/Hc0PTwft](url)